src/config/logger.ts

```diff
@@ -105,6 +105,20 @@ const censorUrl = (url) => {
 	return url;
 };
 
+const headersSerializer = (headers) => {
+	if (!headers) {
+		return headers;
+	}
+
+	const ret = { ...headers };
+	if (ret["Authorization"] || ret["authorization"]) {
+		delete ret["Authorization"];
+		delete ret["authorization"];
+		ret["Authorization"] = "CENSORED";
+	}
+	return ret;
+};
```

Comment:
if you're going to do this, wouldn't it make more sense to "normalize" all headers to be lowercase?